README.md

```diff
@@ -64,6 +64,7 @@
 ## Installation Media
 
 * [nixos-generators](https://github.com/nix-community/nixos-generators) -  Take a NixOS config and build multiple different images types including VirtualBox VMs, Azure images, and installation ISOs.
+* [nixos-infect](https://github.com/elitak/nixos-infect) - Install NixOS on non-NixOS hosts.
```

Comment:
Should probably note:
1. [It's for Linux hosts](https://github.com/elitak/nixos-infect#tested-on)
2. It'll replace their current host, not give them some frankenstein combo of the two.

```suggestion
* [nixos-infect](https://github.com/elitak/nixos-infect) - Replace a running non-NixOS Linux host with NixOS.
```